package.json

```diff
@@ -16,9 +16,10 @@
         "@types/seedrandom": "^3.0.8",
         "@types/webpack": "^5.28.5",
         "@types/webpack-env": "^1.18.8",
-        "@typescript-eslint/eslint-plugin": "^8.24.0",
+        "@typescript-eslint/eslint-plugin": "^8.26.0",
         "@typescript-eslint/parser": "^8.24.0",
         "@vitest/coverage-v8": "^3.0.5",
+        "copy-webpack-plugin": "^13.0.0",
```

Comment:
Do not add new dependencies just for a favicon, there is a simpler way (see other comment)

Reply:
Also, it is important to commit changes to the lock file when you are adding dependencies. (That's the reason the CI failed).